wrappers/mmapwrapper.h

```diff
@@ -144,6 +144,10 @@ namespace HL {
       int fd = -1;
       startAddress = (char *) Alignment;
       mapFlag |= MAP_PRIVATE | MAP_ALIGN | MAP_ANON;
+#elif defined(MAP_ALIGNED)
+      int fd = -1;
+      size_t alignment = (sizeof(size_t) * sizeof(void *)) - 1ul - (size_t)__builtin_clzl(sz);
```

Comment:
This needs some explanatory comments...

Reply:
fair point :)

Reply:
Heap Layers already has an `ilog2` function (defined in `utility/ilog2.h`). I'm assuming this is the same functionality ? If so, please just use that function.

Reply:
ah nice I did not see